src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -171,6 +195,25 @@ trait WorkspaceComponent {
     def unlock(workspaceName: WorkspaceName): ReadWriteAction[Int] = {
       findByNameQuery(workspaceName).map(_.isLocked).update(false)
     }
+
+    //use marsal method
```

Comment:
ignore this. i thought i deleted this comment because i addressed the thing that i was referring to (it was a todo)